### DISCUSSION TITLE
Fastfetch greyed out on first startup

### DISCUSSION THREAD

**Discussion by root9191** [fastfetch-cli/fastfetch#985] [Q&A] [answered]
Hi Guys,

can someone of you tell me, why on first startup from alacritty when fastfetch is loading everything is greyed out?

When i type in fastfetch after that the colours appear :) 
![image](https://github.com/fastfetch-cli/fastfetch/assets/47309435/591d87c6-f174-4a9e-96a5-205df10902f2)

**Comment by CarterLi** [comment, marked as answer]
You should read FAQ in README first

**Reply by root9191** [reply]
Oh Man, i really did but i haven't seen it. Thank you for your help i go to sleep now because i am not productive anymore :smiling_face_with_tear:
